examples/tests/multi_alarm_dont_ignore_previous_overflow/README.md

```diff
@@ -0,0 +1,29 @@
+# Test Multiple Alarms (Ensure previous overflowing alarms aren't ignore)
+
+This tests the virtual alarms available to userspace. It sets three
+alarms alarms. The first two overflow and should result in multiple
+alarms under the hood, and a third a 10 second alarm started after the
+second. The first alarm is set first, but expires second (1 second
+after overflow vs 10ms after overflow). The third alarm should expire
+last.
+
+When successful, the first alarm should fire after a clock overflow +
+1 second, while the third should fire after a clock overflow + 10ms +
+10 seconds.
+
+If the virtual alarm library is buggy, this test might fail by never
+firing the first alarm or firing it after the third alarm
+(specifically almost another clock overflow) because either alarm
+insertion or virtual alarm upcall handing misses it after the second
+alarm fires.
+
+# Example Output
```

Comment:
I suppose? the numbers other than the first column are meaningless except for debugging maybe and they won't be the same across runs because the application boots at slightly different times.

The first column is just the expected order, and should be the same for all boards.

But yes, the nrf5x

Reply:
Added a note, below